cita-executor/core/src/cita_executive.rs

```diff
@@ -123,50 +130,42 @@ impl<B: DB + 'static> CitaExecutive<B> {
         }
 
         let init_gas = t.gas - U256::from(base_gas_required);
-        let _result = match t.action {
+        let result = match t.action {
             Action::Store | Action::AbiStore => {
                 // Maybe use tx_gas_schedule.tx_data_non_zero_gas for each byte store, it is more reasonable.
                 // But for the data compatible, just let it as tx_gas_schedule.create_data_gas for now.
                 let store_gas_used = U256::from(t.data.len() * tx_gas_schedule.create_data_gas);
-                if let Some(gas_left) = init_gas.checked_sub(store_gas_used) {
-                    Ok(TxExecResult { gas_left })
-                } else {
-                    return Err(ExecutionError::VM(VMError::NotEnoughBaseGas));
-                }
+                let mut receipt = Receipt::default();
+                receipt.quota_used = store_gas_used.as_u64();
+                Ok(receipt)
             }
             Action::Create => {
-                // Call cita-vm interface.
-                //                let account_address = &H160::from(t.sender().into_fixed_bytes());
-                //                let nonce = match self.state_provider.borrow_mut().get_state_object(account_address) {
-                //                    Ok(opt_account) => {
-                //                        if let Some(account) = opt_account {
-                //                            account.nonce
-                //                        } else {
-                //                            U256::zero()
-                //                        }
-                //                    }
-                //                    Err(e) => {
-                //                        log::error!(target: "evm executor", "{}", e);
-                //                        U256::zero()
-                //                    }
-                //                };
-                //                let evm_transaction = build_evm_transaction(&t, nonce);
-                //                if cita_vm::exec(
-                //                    self.block_provider,
-                //                    self.state_provider,
-                //                    evm_context,
-                //                    evm_config,
-                //                    evm_transaction,
-                //                ).is_ok() {
-                //
-                //                }
-                Ok(TxExecResult { gas_left: init_gas })
+                let params = VmExecParams {
+                    code_address: None,
+                    sender,
+                    to_address: None,
+                    gas: init_gas,
+                    gas_price: t.gas_price(),
+                    value: t.value,
+                    nonce: U256::zero(),
+                    data: t.data.clone(),
+                };
+
```

Comment:
We should get params by re-translating transaction info.

Reply:
Actually, we have done this. And I just need only this params to create a Contract.